haystack/agents/utils.py

```diff
@@ -2,7 +2,7 @@
 
 from haystack.agents.types import Color
 
-STREAMING_CAPABLE_MODELS = ["davinci"]
+STREAMING_CAPABLE_MODELS = ["davinci", "gpt-3.5", "gpt-35-turbo", "gpt-4"]
```

Comment:
Isn't the model name `gpt-3.5-turbo` instead of `gpt-3.5`?

Reply:
it will catch it the same way it uses "davinci" for catching "text-embedding-davinci-003"

Reply:
Yeah true @recrudesce but we agreed to spell out the names for clarity.